first-steps.mdx

```diff
@@ -18,25 +18,40 @@ The second page will render a screenshot of the url you provided. It tests that
 Now you'd need to give your project a name. We'll propose one similar to the url you provided.
 But you can choose your own.
 
-<Frame caption="Second page of the setup flow: Name your project, screenshot 08/2023">
+<Frame caption="Second page of the setup flow: Name your project, screenshot 09/2023">
   <img
     src="/images/setup/setup-2-project.png"
-    alt="name your project, screenshot 08/2023"
+    alt="name your project screen, screenshot 09/2023"
   />
 </Frame>
 
 <div class="mt-8" />
 
-## 3. Give us a test user
+## 3. Select UI test cases to be auto-discovered
 
-Almost all web apps are featuring a sign-in and interesting test cases can only be executed after sign-in. This is why
-we need a test user. Also, the first test case we will try to discover will be a sign-in with a username and a password.
+At the first set-up, you can select if we should auto-discover:
+
+- a `log in` test for your web app
+- a `change language` test
+- a reachability `check for all links` on your page
+- or just go with only custom test cases, where you can create test cases:
+  - with our `AI agent` \(alpha feature\)
+  - with a `Playwright recording` tool
+
+<Frame caption="Third page of the setup flow - choose your first tests, screenshot 10/2023">
+  <img
+    src="images/setup/setup-3-custom.png"
+    alt="choose your first tests screen, screenshot 10/2023"
+  />
+</Frame>
+
+If you choose the `log in` test, we'll ask you for **test user credentials** \(test username and a password\).
 The test credentials should fit the URL you provided above.
 
-<Frame caption="Third page of the setup flow - test user credentials, screenshot 08/2023">
+<Frame caption="Third page of the setup flow - insert test user credentials for a log-in test, screenshot 10/2023">
   <img
     src="/images/setup/setup-3-credentials.png"
```

Comment:
<img width="534" alt="Screenshot 2023-10-24 at 13 50 36" src="https://github.com/OctoMind-dev/mintlify-docs/assets/22753474/4f385daa-c576-4fbc-9336-dd5bccae97a8">
 
i don't like the removed text on the FAB ;)

Reply:
I didn't figure out how to remove it entirely. When you make screenshots it's often in the way of things needed to be seen.